mne/viz/_3d.py

```diff
@@ -289,11 +290,15 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         It corresponds to Freesurfer environment variable SUBJECTS_DIR.
     ch_type : None | 'eeg' | 'meg'
         This argument is deprecated. Use meg_sensors and eeg_sensors instead.
-    source : str
-        Type to load. Common choices would be `'bem'` or `'head'`. We first
-        try loading `'$SUBJECTS_DIR/$SUBJECT/bem/$SUBJECT-$SOURCE.fif'`, and
-        then look for `'$SUBJECT*$SOURCE.fif'` in the same directory. Defaults
-        to 'bem'. Note. For single layer bems it is recommended to use 'head'.
+    source : str | list
+        Type to load. Common choices would be `'bem'`, `'head'` or
+        `'outer_skin'`. If list, the sources are looked up in the given order
+        and first found surface is used. We first try loading
+        `'$SUBJECTS_DIR/$SUBJECT/bem/$SUBJECT-$SOURCE.fif'`, and then look for
+        `'$SUBJECT*$SOURCE.fif'` in the same directory. For `'outer_skin'`,
+        `'outer_skull'` and `'inner_skull'`, the subjects bem/flash folder is
+        searched. Defaults to 'bem'. Note. For single
+        layer bems it is recommended to use 'head'.
```

Comment:
I thought it would be a good idea to allow different surfaces by this param, but the lookup procedure ended up being quite magical. Not sure what would be the best way of doing this. Wdyt?